src/commands/informatics/removewebhook.js

```diff
@@ -0,0 +1,36 @@
+const fetch = require("node-fetch")
+const { apiurl } = require("../../../config.json")
+
+
+module.exports = {
+    config: {
+        name: "removewebhook",
+        aliases: [],
+        usage: "<webhook ID> <webhook token>",
+        category: "informatics",
+        description: "Removes a webhook from recieving FAGC notifications",
+        accessibility: "Moderator",
+    },
+    run: async (client, message, args) => {
+        if (!args[0]) return message.reply("Provide a Webhook ID")
+        if (!args[1]) return message.reply("Provide a Webhook token")
```

Comment:
That is the webhook token, I use it because it is much longer and more random, similar format to bot API keys. Mainly for security purposes

Reply:
I see, you are using the token as a way to authorise the removal, rather than using an api key.

Reply:
Yep. The way to authorize webhook removal is API key, guild ID, webhook ID and webhook token. Might be a but much, but it makes sure that nothing gets accidentally deleted or any unauthorized access.

Reply:
Sorry, I forgot why I did it like this and how it works. You were correct, this was done this way because even guilds without an API key would want to be able to listen to events with webhooks and be able to remove them. So yes, I use a token to authorise the removal, rather than using an API key.

Reply:
Ok that is fine, just needed to know why a token was required for removal.